Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -134,9 +141,7 @@ - (SNTFileInfo *)fileInfo {
   if (!_fileInfo) {
     _fileInfo = [[SNTFileInfo alloc] initWithPath:self.filePath];
     if (!_fileInfo) {
-      if (isatty(STDOUT_FILENO) && !self.jsonOutput) {
-        printf("\rInvalid or empty file: %s\n", self.filePath.UTF8String);
-      }
+      if (PrettyOutput()) printf("\rInvalid or empty file: %s\n", self.filePath.UTF8String);
```

Comment:
Sending it to stderr. Still need the \r for formatting when viewing stdout and stderr together.